Source/CombatExtended/CombatExtended/Jobs/Utils/JobGiverUtils_Reload.cs

```diff
@@ -147,6 +147,13 @@ public static bool CanReload(Pawn pawn, Thing thing, bool forced = false, bool e
             JobFailReason.Is("CE_TurretNonAllied".Translate());
             return false;
         }
+
+        if (!pawn.health.capacities.CapableOf(PawnCapacityDefOf.Manipulation))
+        {
+            CELogger.Message($"{pawn} could not reload {turret} because it's not capable of manipulation.");
```

Comment:
We could probably replace "it's" with the pawn's pronoun--just can't recall exactly how to key that.